src/middleware/log_request.rs

```diff
@@ -33,12 +33,13 @@ impl Handler for LogRequests {
 
         print!(
             "at={level} method={method} path=\"{path}\" \
-             request_id={request_id} fwd=\"{ip}\" service={time_ms}ms \
+             request_id={request_id} fwd=\"{ip}\" real_ip=\"{real_ip}\" service={time_ms}ms \
              status={status} user_agent=\"{user_agent}\" referer=\"{referer}\"",
             level = level,
             method = req.method(),
             path = FullPath(req),
             ip = request_header(req, "X-Forwarded-For"),
+            real_ip = req.remote_addr(),
```

Comment:
This is going to be nginx connecting from `127.0.0.1`.  We still need to use the last value in `X-Forwarded-For` from the app.

Reply:
We could also add `proxy_set_header X-Real-IP $remote_addr;` to nginx if we want to pass it's interpretation along to the app.

Reply:
Ugh for some reason I thought that nginx would do some magic here. I like your X-Real-Ip idea.